src/layouts/Layout.js

```diff
@@ -1,32 +1,18 @@
-import React from "react";
 import Header from "./Header";
 import Content from "./Content";
 import Footer from "./Footer";
-// import { makeStyles } from "@material-ui/core/styles";
-import Grid from "@material-ui/core/Grid";
-
-// const useStyles = makeStyles((theme) => ({
-//   root: {
-//     display: "flex",
-//     flexDirection: "row",
-//     justifyContent: "space-between",
-//     flexDirection: "column",
-//     backgroundColor: "#343B41",
-//     color: "#F7F7F7",
-//     height: "40%",
-//   },
-// }));
-
-function Layout() {
-  // const classes = useStyles();
+import { BrowserRouter as Router, Switch, Route, Link } from "react-router-dom";
 
+const Layout = () => {
   return (
-    <Grid>
-      <Header />
-      <Content />
-      <Footer />
-    </Grid>
+    <>
+      <Router>
```

Comment:
I know this is basic setup - you added Link's in Navbar/Footer, but there are nowhere Route's declarations so Links will not work:
https://reactrouter.com/web/example/basic

Good practice is to make const with routes, like:

```
export const paths = {
  PAGE_1: "/page-1",
  PAGE_2: "/page-2",
}

and to make Routes:
<Router>
<Header />
 <Switch>
    <Route  exact path={paths.PAGE_1} component={Page1} />
    etc
  </Switch>
 </Router>
 <Footer />

where Content will be one of thee Route and component on that route

Reply:
Noted on export const paths and routes. I plan adding them in other branch. With main content container.